ReadingList.md

```diff
@@ -0,0 +1,9 @@
+# Reading List
+
+This is a collection of books and articles to continue learning beyond this repo.
+Most of these are suggestions by Kevin right now, but if you have something feel free to add it.
+
+- Head First Design Patterns
+- The Pragmatic Programmer - Thomas & Hunt
```

Comment:
- [The Pragmatic Programmer - Thomas & Hunt](https://www.amazon.com/Pragmatic-Programmer-Journeyman-Master/dp/020161622X)